src/main/java/com/github/manolo8/darkbot/utils/StartupParams.java

```diff
@@ -84,24 +89,8 @@ public StartupParams(String[] args) throws IOException {
         }
     }
 
-    /* Auto login */
-    private static Properties loadLoginProperties(String path) throws IOException {
-        Properties p = new Properties();
-        try (InputStreamReader reader = new InputStreamReader(new FileInputStream(path), StandardCharsets.UTF_8)) {
-            p.load(reader);
-        }
-        System.out.println("Loaded startup properties file");
-        return p;
-    }
-
-    public String getAutoLoginValue(PropertyKey key) {
-        Properties p = (Properties) startupParams.get(LaunchArg.LOGIN);
-        return p.getProperty(key.name().toLowerCase(Locale.ROOT));
-    }
-
-    public char[] getAutoLoginMasterPassword() {
-        String masterPassword = getAutoLoginValue(PropertyKey.MASTER_PASSWORD);
-        return masterPassword == null ? null : masterPassword.toCharArray();
+    public AutoLoginProps getAutologinInstance(){
```

Comment:
```suggestion
    public AutoLoginProps getAutoLogin() {
```
formatting

Reply:
Method with same name already exist
```
public boolean getAutoLogin() {
    return has(LaunchArg.LOGIN);
}
```

Reply:
I was mostly concerned about capitalization and the space, you could name it `getAutoLoginProps()` too